tests/test_vrt.py

```diff
@@ -3,19 +3,10 @@
 import subprocess
 import jsonschema
 import os
-import glob
-
 
 class TestVrt(unittest.TestCase):
     def setUp(self):
-        self.mappings = [
-            {
-                'filename': f,
-                'name': os.path.splitext(os.path.basename(f))[0]
-            }
-            for f in glob.glob(utils.MAPPING_DIR + '/*.json')
-            if 'schema' not in f
-        ]
+        self.mappings = utils.make_mapping('/*' + '/*.json', True) + utils.make_mapping('/*.json', False)
```

Comment:
Is this intentional? It used to be just `/*.json`. If it's meant to be `/*/*.json` why not just making a single string instead of the concat?

Reply:
I'd like the test to support both versions so that if something changes we know.  The problem with this was that when we moved the mappings no tests broke.

Reply:
I realize that's not what you meant.  ya I'll update it.